### ISSUE TITLE
Time-varying covariate

### ISSUE THREAD

**Issue by sardnar** [pharmpy/pharmpy#2523]
### Feature request

Hello,
I am not sure if there is a way to specify a time-varying covariate on parameter(s). 

Thank you!

**Comment by rikardn** [comment]
Currently not. Are you thinking about how the scm in PsN is handling time-varying covariates or something else?

**Comment by sardnar** [comment]
Yes, just another type of covariates. Is there is any difference between scm and pharmpy covsearch anyway?!

**Comment by rikardn** [comment]
The `scm` in PsN  and `covsearch` per default handles time-varying covariates in the same way. If a reference value is needed for example a median over the individuals this is calculated as the median over the median over the individuals. There is an alternative way in `scm` to do this calculation for time-varying covariates via the `time_varying` option. `covsearch` does not (yet) support this.

**Comment by sardnar** [comment]
@rikardn I just have one question related here. It seems that scm will require all parameters to be TVpar labelled, but pharmpy does not use this notation that will make scm fails. Is there is a reason for this discrepancy?

**Comment by rikardn** [comment]
PsN needs the extra guidance from the user to be able to add the covariate effect correctly. Pharmpy is able to figure this out without the explicit `TVpar`. This in turn because Pharmpy is parsing all expressions in the NONMEM code into symbolic statements that can be manipulated or "understood" programmatically.
